requirements.txt

```diff
@@ -35,6 +35,10 @@ pyvista>=0.32,!=0.35.2
 pyvistaqt>=0.4
 mffpy>=0.5.7
 ipywidgets
+ipympl; platform_system!="Windows"  # This causes core mne viz tests to fail!
+									# XXX: Add support when ipympl CIs
+									# include Windows
```

Comment:
Nitpick, but can you move these all to one long line? It keeps `requirements.txt` more readable I think (and we don't care about 80 char limit in this file)